### ISSUE TITLE
Some doubts

### ISSUE THREAD

**Issue by ParasharaRamesh** [giacomolamberti90/CS229_project#2]
Hello guys,
So I have been trying to replicate and better your model by adding attention and also self attention.
So as of now I have been able to replicate your gru model with vgg in Google colab with tensorflow backend as opposed to theano.
So my doubts are as follows:
1. After running 50 epochs what was your models loss value ?( i.e. your models best loss value as shown in keras when you try to fit the data) 
2. Over how  many caption,image pairs have you trained it over?

I have tried triaining the model with data of over 16000 image,caption pairs and the best loss value I have is about 2.5 ish.

So could you please tell me at what point did your loss come really close to 0?
Thanks
